src/containers/access/src/reducers/updateAttachmentTagReducer.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+
+import { Map } from 'immutable';
+import { ReduxConstants } from 'lattice-utils';
+import { RequestStates } from 'redux-reqseq';
+import type { SequenceAction } from 'redux-reqseq';
+
+import {
+  UPDATE_ATTACHMENT_TAG,
+  updateAttachmentTag,
+} from '../actions';
+
+const { REQUEST_STATE } = ReduxConstants;
+
+export default function updateAttachmentTagReducer(state :Map<*, *>, action :SequenceAction) {
+
+  return updateAttachmentTag.reducer(state, action, {
+    REQUEST: () => {
+      const { path, tag } = action.value;
+      return state
+        .setIn(path, tag)
+        .setIn([UPDATE_ATTACHMENT_TAG, REQUEST_STATE], RequestStates.PENDING);
```

Comment:
It's fine to update the store with the new value before the request finishes, but you're not handling the failure case.

Reply:
yeah i feel like the failure case should yield a snackbar but not sure yet. reverting back to previous tag doesn't seem right. just leaving for now.

Reply:
I like the idea of toasts, let's work with Shireen to standardize usage.